### ISSUE TITLE
Stoki / Krond / Gromma Gold Chest contents differ

### ISSUE THREAD

**Issue by QuickMythril** [Nomad7/ICCI-list#6]

**Comment by QuickMythril** [comment]
History:
Barbarian - 12/25/2018:
Stoki, Krond, Gromma event chests are different - same odds like old gold chests.  they said they might make them the same after Stoki's event but at the end didn't.  that's when I noticed the difference and filed a ticket.

when the first YR2 event came around I noticed the difference because I was desperately hunting for potions which were not falling from Farideh's chests so I noticed the difference.

the support then explained to me that probably they will make Krond and Gromma's chest (and Stoki, retroactively) the same as the newer champs for future, but it didn't happen in the next two events.
